### ISSUE TITLE
I'm training loras for wan1.3B. It's very fast and doesn't work in comfyUI even with the conversion. It doesn't give an error message. 16 epochs. I've already put up to 100 epochs. What would be the error?

### ISSUE THREAD

**Issue by ajrss2009** [kohya-ss/musubi-tuner#140]
I'm training loras for wan1.3B. It's very fast and doesn't work in comfyUI even with the conversion. It doesn't give an error message. 16 epochs. I've already put up to 100 epochs. What would be the error?

python.exe wan_cache_latents.py --dataset_config D:\PYTHONS\LORA_TRAINER\musubi-tuner\train\Thanos.TOML --vae D:\PYTHONS\LORA_TRAINER\musubi-tuner\wan\models\wan_2.1_vae.safetensors --vae_cache_cpu

python.exe wan_cache_text_encoder_outputs.py --dataset_config D:\PYTHONS\LORA_TRAINER\musubi-tuner\train\Thanos.TOML --t5 D:\PYTHONS\LORA_TRAINER\musubi-tuner\wan\models\models_t5_umt5-xxl-enc-bf16.pth

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 wan_train_network.py --task t2v-1.3B --dit D:\\PYTHONS\\LORA_TRAINER\\musubi-tuner\\wan\\models\\wan2.1_t2v_1.3B_fp16.safetensors --dataset_config D:\\PYTHONS\\LORA_TRAINER\\musubi-tuner\\train\\Thanos.TOML --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 5e-5 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 3.0 --max_train_epochs 220 --save_every_n_epochs 1 --seed 42 --output_dir D:\\PYTHONS\\LORA_TRAINER\\musubi-tuner\\loras\\Wan2.1\\ --output_name Thanos_Wan
